packages/server/s3storage.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
       const s3upload = multer({
         storage: multerS3({
           s3: createS3Client(),
-          bucket: getState().getConfig("storage_s3_bucket"),
+          bucket: getState().getConfig("storage_s3_bucket") ?? "emlatech",
```

Comment:
remove default here, if it is omitted we should show the error message and crash like i think you do in the code